src/networkrequest.cpp

```diff
@@ -92,7 +92,7 @@ QString NetworkRequest::apiBaseUrl() {
   if (pe.contains("MVPN_API_BASE_URL")) {
```

Comment:
This part will overwrite the staging server value. I would like to see:
1. remove this `MVPN_API_BASE_URL` check here and simply return: `Constants::getStagingServerAddress()`.
2. see the other comment in settingsholder.cpp